example_test.go

```diff
@@ -133,32 +131,32 @@ func Example_openTelemetryLog() {
 	// {"component":"github.com/go-kod/kod/examples/helloworld/HelloWorld","level":"INFO","msg":"Hello, World!"}
 }
 
-// This example demonstrates how to use tracing with OpenTelemetry.
-func Example_openTelemetryTrace() {
-	logger, observer := kod.NewTestLogger()
-
-	kod.Run(context.Background(), func(ctx context.Context, app *helloworld.App) error {
-		ctx, span := app.Tracer().Start(ctx, "example")
-		defer span.End()
-		app.L(ctx).Info("Hello, World!")
-		app.L(ctx).WarnContext(ctx, "Hello, World!")
-
-		app.HelloWorld.Get().SayHello(ctx)
-		return nil
-	}, kod.WithInterceptors(ktrace.Interceptor()), kod.WithLogger(logger))
-
-	fmt.Println(observer.Filter(func(m map[string]any) bool {
-		return m["trace_id"] != nil && m["span_id"] != nil
-	}).RemoveKeys("trace_id", "span_id", "time"))
-
-	// Output:
-	// helloWorld init
-	// Hello, World!
-	// helloWorld shutdown
-	// {"component":"github.com/go-kod/kod/Main","level":"INFO","msg":"Hello, World!"}
-	// {"component":"github.com/go-kod/kod/Main","level":"WARN","msg":"Hello, World!"}
-	// {"component":"github.com/go-kod/kod/examples/helloworld/HelloWorld","level":"INFO","msg":"Hello, World!"}
-}
+// // This example demonstrates how to use tracing with OpenTelemetry.
+// func Example_openTelemetryTrace() {
```

Comment:
commented code like this always looks suspicious

Either it's something left behind during refactoring that need to be addressed and fixed, either it should be removed

Reply:
You're right, this should be removed :-)
I'm planning simplify kod's core logic, and move OpenTelemetry related code to github.com/go-kod/kod-ext

Reply:
Kodext or kodex could be a better name as it sounds like codex

Also hyphen in a go package is never fun, except the classic go-whatever